<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBare</class>
 <widget class="QWidget" name="DialogBare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>485</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DialogBare</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="50">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayoutBrowser" columnstretch="1,0,0,0,0,0,0,0,0,0">
       <item row="0" column="5">
        <widget class="QComboBox" name="boardComboBox"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="portLabel">
         <property name="text">
          <string>Port:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="10">
        <widget class="QPlainTextEdit" name="upgradeLog">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="maximumBlockCount">
          <number>800</number>
         </property>
        </widget>
       </item>
       <item row="2" column="9">
        <widget class="QPushButton" name="cancelButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QPushButton" name="selectFileButton">
         <property name="text">
          <string>Select File</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QComboBox" name="versionComboBox">
         <item>
          <property name="text">
           <string extracomment="Recommended for normal users">Stable Version</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Beta Version</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nightly Build</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="boardIdLabel">
         <property name="text">
          <string>Board</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QComboBox" name="portBox">
         <item>
          <property name="text">
           <string>&lt;Automatic&gt;</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="advancedCheckBox">
         <property name="text">
          <string>Advanced</string>
         </property>
        </widget>
       </item>
       <item row="0" column="9">
        <widget class="QPushButton" name="flashButton">
         <property name="text">
          <string>Flash</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QPushButton" name="scanButton">
         <property name="text">
          <string>Scan</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="9">
        <widget class="QProgressBar" name="upgradeProgressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
